etcdctl/README.md

```diff
@@ -1082,6 +1082,12 @@ echo ${transferee_id}
 
 LOCK acquires a distributed mutex with a given name. Once the lock is acquired, it will be held until etcdctl is terminated.
 
+#### Example
+
+```bash
+etcdctl role grant-permission my_rw_locks --prefix=true readwrite _locks//my_service_locks/
```

Comment:
@alwindoss If I understood correctly `my_rw_locks` treated here as a name of role and `_locks//my_service_locks/` as key, here is how the role command looks like `role grant-permission [options] <role name> <permission type> <key> [endkey] [flags]` . How the locks relevant here?   Isn't this proposed command here similar to the examples already given here - https://github.com/etcd-io/etcd/blob/f1e2c22139461bcc218470cfa54b374db7181d5d/etcdctl/README.md#examples-16 ? I might be misunderstanding it, but in that case, it would be great if the command is documented here clearly with more details. Also, this example should not be added here but at the end of lock or role section. Please also take a look at this discussion https://github.com/etcd-io/etcd/pull/10757 Thanks!